boards/arc/em_starterkit/Kconfig.board

```diff
@@ -8,6 +8,7 @@ config BOARD_EM_STARTERKIT
 	bool "ARC EM Starter Kit"
 	depends on SOC_EMSK
 	select HAS_DTS
+	select HAS_I2C_DW
```

Comment:
Seems like this should really be in SoC not board.

Reply:
Fixed.